scripts/start.sh

```diff
@@ -7,8 +7,45 @@ set -e
 ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cd "$ScriptDir"
 
+# get the port from the REACT_APP_BACKEND_URI env variable
+envContent=$(grep -v '^#' ../webapp/.env | xargs)
+backendPort=$(echo $envContent | sed -n 's/.*:\([0-9]*\).*/\1/p')
+
 # Start backend (in background)
 ./start-backend.sh &
 
-# Start frontend
-./start-frontend.sh
+# set max retries to 5
+maxRetries=5
+
+# set retry count to 0
```

Comment:
This comment should be removed: it just repeats what's self evident from the code